lib/oidcstrategy.js

```diff
@@ -143,6 +143,22 @@ function Strategy(options, verify) {
       This is fine if you are expecting multiple organizations to connect to your app.
       Otherwise you should validate the issuer.`);
   }
+
+  // validate other necessary option items provided, we validate them here and only once
+  var itemsToValidate = objectTransform({
+    source: options,
+    pick: ['clientID', 'callbackURL', 'responseType', 'responseMode']
+  });
+  var validatorConfiguration = {
+    clientID: Validator.isNonEmpty,
+    callbackURL: Validator.isNonEmpty,
+    // callbackURL: Validator.isURL  // We allow http for now
```

Comment:
Please do allow us to keep using HTTP for local dev scenarios without having to do a TLS dance; but a configuration property is ideal - for live dev, test and production scenarios we'd want to make sure that is off.

Real case is: give me `localhost:3000` please :).